cmd/gator/test/test.go

```diff
@@ -154,3 +209,31 @@ func errFatalf(format string, a ...interface{}) {
 	fmt.Fprintf(os.Stderr, format, a...)
 	os.Exit(1)
 }
+
+func statsToYAMLString(stats []*instrumentation.StatsEntry) string {
```

Comment:
It appears these can go away once the "remove write to separate file functionality" comment is addressed.